docs/src/token-2022/extensions.mdx

```diff
@@ -1763,3 +1763,157 @@ Coming soon!
 
   </TabItem>
 </Tabs>
+
+### Group Pointer
+
+Similar to the metadata pointer, the group pointer allows a token creator to
+designate a group account that describes the mint. However, rather than
+describing token metadata, the group account describes configurations for
+grouping tokens together.
+
+When a Token-2022 mint possesses a group pointer, the mint is considered to be
+a group mint (for example a Collection NFT). Group mints have configurations
+that allow them to be used as a point of reference for a related set of tokens.
+
+Similar to metadata, the group pointer can point to the mint itself, and a
+client must check that the mint and the group both point to each other.
+
+#### Example: Create a mint with a group pointer to an external account
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+$ spl-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb create-token --group-address 7ZJVSav7y76M41eFeyA3xz39UDigQspVNwyJ469TgR1S
+Creating token EUMhJgfvjZa7Lb7fSqfD6WCUwELzzRVKunKSnSi4xK42 under program TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb
+
+Address:  EUMhJgfvjZa7Lb7fSqfD6WCUwELzzRVKunKSnSi4xK42
+Decimals:  9
+
+Signature: 3ug4Ejs16jJgEm1WyBwDDxzh9xqPzQ3a2cmy1hSYiPFcLQi9U12HYF1Dbhzb2bx75SSydfU6W4e11dGUXaPbJqVc
+```
+
+  </TabItem>
+  <TabItem value="jsx" label="JS">
+
+Coming soon!
+
+  </TabItem>
+</Tabs>
+
+### Group
+
+Token-2022 supports grouping of tokens through the group extension. The
+configurations for a group, which describe things like the update authority
+and the group's maximum size, can be stored directly in the mint itself.
+
+Token-2022 implements all of the instructions from the
+[spl-token-group-interface](https://github.com/solana-labs/solana-program-library/tree/master/token-group/interface).
+
+The group extension should work directly with the group-pointer extension.
+During mint creation, you should also add the group-pointer extension, pointed
+at the mint itself.
```

Comment:
I believe this is enforced during group creation, right? If so, we can use stronger language: "you must add the group-pointer extension"